apps/api/src/app/routes/chains/__chainId/markets/__baseTokenAddress-__quoteTokenAddress/slippageTolerance.ts

```diff
@@ -11,16 +11,15 @@ import {
   getCacheControlHeaderValue,
 } from '../../../../../../utils/cache';
 
+const CACHE_SECONDS = 120;
+
 // TODO:  Add this in a follow up PR
 // import { ALL_SUPPORTED_CHAIN_IDS } from '@cowprotocol/cow-sdk';
 
-interface Result {
-  slippageBps: number;
-}
-
 const routeSchema = {
   type: 'object',
   required: ['chainId', 'baseTokenAddress', 'quoteTokenAddress'],
+  additionalProperties: false,
```

Comment:
With this the schema now accepts unknown properties.
What kind of properties do you expect?

Reply:
just the opposite, I don't accept them. I did this, to make sure the inferred type was what I was expecting